runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -5855,6 +5853,10 @@ class INTERPRETER_CLASS
 					returnVoidFromINL(REGISTER_ARGS, 4);
 				}
 			}
+
+			if (immediateAsyncPending()) {
+				rc = GOTO_ASYNC_CHECK;
+			}
```

Comment:
I think this is the first time this has been called after an INL return (which does not return, it just adjusts the stack). Offhand, I'm not sure this is incorrect, but may be worth thinking about.

Reply:
Actually, I think this is unnecessary for the same reason as above, being in a native.